--- gettext-runtime/gnulib-lib/stdint.in.h.orig	Mon May 24 09:42:46 2010
+++ gettext-runtime/gnulib-lib/stdint.in.h	Tue May 31 16:08:07 2011
@@ -478,10 +478,10 @@
    This include is not on the top, above, because on OSF/1 4.0 we have a sequence of nested
    includes <wchar.h> -> <stdio.h> -> <getopt.h> -> <stdlib.h>, and the latter includes
    <stdint.h> and assumes its types are already defined.  */
-#if ! (defined WCHAR_MIN && defined WCHAR_MAX)
-# define _GL_JUST_INCLUDE_SYSTEM_WCHAR_H
-# include <wchar.h>
-# undef _GL_JUST_INCLUDE_SYSTEM_WCHAR_H
+#if ! (defined WCHAR_MIN && defined WCHAR_MAX) && ! (defined _MINIX)
+#define _GL_JUST_INCLUDE_SYSTEM_WCHAR_H
+#include <wchar.h>
+#undef _GL_JUST_INCLUDE_SYSTEM_WCHAR_H
 #endif
 #undef WCHAR_MIN
 #undef WCHAR_MAX
