--- gettext-tools/libgrep/m-fgrep.c	Mon May 30 20:47:53 2011
+++ gettext-tools/libgrep/m-fgrep.c	Mon May 30 20:54:10 2011
@@ -30,6 +30,13 @@
 #if defined HAVE_WCTYPE_H && defined HAVE_WCHAR_H && defined HAVE_MBRTOWC
 /* We can handle multibyte string.  */
 # define MBS_SUPPORT
+#endif
+
+#ifdef __minix
+#undef MBS_SUPPORT
+#endif
+
+#if defined MBS_SUPPORT
 # include <wchar.h>
 # include <wctype.h>
 #endif
